application.py

```diff
@@ -14,7 +16,7 @@
 }
 
 manager = Manager(application)
-manager.add_command("runserver", Server(port=5003))
+manager.add_command("runserver", Server(host='0.0.0.0', port=port))
```

Comment:
`python application.py runserver` accepts port and host as options, so we can set them in Procfile instead and there's no need for a custom ENV variable